packages/components/src/list/index.js

```diff
@@ -2,119 +2,54 @@
  * External dependencies
  */
 import classnames from 'classnames';
-import { Component } from '@wordpress/element';
-import { ENTER } from '@wordpress/keycodes';
 import PropTypes from 'prop-types';
 import { CSSTransition, TransitionGroup } from 'react-transition-group';
 
 /**
  * Internal dependencies
  */
-import Link from '../link';
+import ListItem from './list-item';
```

Comment:
Moved most of this to the `ListItem` component, allowing us to just use the `List` component as a stand alone with custom children (see readme example).